src/shogun/machine/Composite.h

```diff
@@ -78,7 +78,7 @@ namespace shogun
 				}, v.second);
 			}
 			m_ensemble_machine->train(current_data, labs);
-			return m_ensemble_machine;
+			return true;
```

Comment:
@LiuYuHui I had to change the return type, since that is the type of Machine::train. In future it might be worth returning the Machine object.